pallets/proposals/src/impls.rs

```diff
@@ -172,6 +172,13 @@ impl<T: Config> Pallet<T> {
                 p.withdrawn_funds = p.withdrawn_funds.saturating_add(withdrawable);
                 if p.withdrawn_funds == p.raised_funds {
                     <T as Config>::DepositHandler::return_deposit(p.deposit_id)?;
+                    // TODO: reinstate storage deposit
+                    CompletedProjects::<T>::try_mutate(project_account, |completed_projects| -> DispatchResult {
+                        if !completed_projects.iter().any(|&completed_project| completed_project == project_key) {
```

Comment:
you can just use Iterator::contains() here

Reply:
Or i wonder wether we should save the computation and dont bother checking wether its already in the vector. 

We can assume that this will only be called once safely because the project is being deleted a couple lines later.
Also it will make our benchmarking simpler

Reply:
Sure